llmfoundry/models/mpt/configuration_mpt.py

```diff
@@ -7,6 +7,8 @@
 
 from transformers import PretrainedConfig
 
+from llmfoundry.models.utils import is_torch_2_or_higher
```

Comment:
im actually not entirely sure why, but i think changing this to `from llmfoundry.models.utils.misc import is_torch_2_or_higher` will fix your test failure. the way huggingface does imports is a bit tricky.